pkg/apic/provisioning/applicationrequest.go

```diff
@@ -2,10 +2,10 @@ package provisioning
 
 // ApplicationRequest - interface for agents to use to get necessary application request details
 type ApplicationRequest interface {
+	// GetAgentDetailsValue return the value based on the key
+	GetAgentDetailsValue(key string) string
```

Comment:
To keep consistent with the others we should change this to GetApplicaitonDetailsValue

Reply:
Yes